src/main/java/com/epam/service/JwtService.java

```diff
@@ -0,0 +1,73 @@
+package com.epam.service;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
+public class JwtService {
+    private static final String SECRET_KEY = "jJKtCS6NA1sIY1S4iRaqkGF6rwpfCrFrWVqzj38r2BY=";
```

Comment:
Please get rid of this secret. You need to leverage a keystore and save keys there.